react/components/BaseProductIdentifier.tsx

```diff
@@ -27,25 +28,33 @@ const messages = defineMessages({
   },
 })
 
+const CSS_HANDLES = [
+  'productIdentifier',
+  'productIdentifierLabel',
+  'productIdentifierSeparator',
+  'productIdentifierValue'
+]
+
 const ProductIdentifier: StorefrontFunctionComponent<
   ProductIdentifierProps & InjectedIntlProps
 > = ({ idField, value, customLabel, label, intl }) => {
+  const handles = useCssHandles(CSS_HANDLES)
   return (
     <span
-      className={`${styles['product-identifier']} ${styles['product-identifier']}--${idField} c-muted-1`}
+      className={`${handles.productIdentifier} ${handles.productIdentifier}--${idField} c-muted-1`}
     >
       {label !== 'hide' && (
-        <span className={styles['product-identifier__label']}>
+        <span className={`${handles.productIdentifierLabel}`}>
           {label === 'custom' && <IOMessage id={customLabel} intl={intl} />}
           {label === 'default' && (
             <FormattedMessage id={messages[idField].id} />
           )}
         </span>
       )}
       {label !== 'hide' && (
-        <span className={styles['product-identifier__separator']}>: </span>
+        <span className={`${handles.productIdentifierSeparator}`}>: </span>
       )}
-      <span className={styles['product-identifier__value']}>{value}</span>
+      <span className={`${handles.productIdentifierValue}`}>{value}</span>
```

Comment:
If you're wondering how you could access object keys that are strings, you could just do:

```typescript
handles['product-identifier__label']
```